### PR TITLE
DS-41 power spectrum ui fix

### PR DESCRIPTION
**Pull request by AgustinSF**

### CODE REVIEW

**Review by TommyWoodley — APPROVED** [review]
LGTM

File: src/waveview/wave.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 class RootWave(BoxLayout):
     sample_rate = 44100
     num_samples = 44100
-    time = 5
+    time = 2
```
Comment (TommyWoodley) [inline src/waveview/wave.py:16]: Why 2s? Just for now to reduce delay?

Comment (AgustinSF) [reply src/waveview/wave.py:16]: I was trying to fix the x-coordinates of the waveform graph, but now that I think of it, this doesn't fix it.

Comment (TommyWoodley) [reply src/waveview/wave.py:16]: Is it broken atm?

Comment (AgustinSF) [reply src/waveview/wave.py:16]: Not broken but wrong

Comment (TommyWoodley) [reply src/waveview/wave.py:16]: ok. maybe merge this PR. Cause I think all the other changes are good. And then make another ticket for investigating why it's wrong

Comment (TommyWoodley) [reply src/waveview/wave.py:16]: And I think 2s is actually good cause that's the time period we agreed on mapping (with you being able to change the period to the next 2s)

**Review by DCupello1 — APPROVED** [review]

**Pull request merged**